modulemd/v2/modulemd-module.c

```diff
@@ -389,6 +389,28 @@ modulemd_module_add_stream (ModulemdModule *self,
 }
 
 
+GStrv
+modulemd_module_get_stream_names_as_strv (ModulemdModule *self)
+{
+  g_return_val_if_fail (MODULEMD_IS_MODULE (self), NULL);
+
+  if (!self->streams)
+    return NULL;
+
+  g_autoptr (GHashTable) stream_names =
+    g_hash_table_new (g_str_hash, g_str_equal);
+
+  for (guint i = 0; i < self->streams->len; i++)
+    {
+      g_hash_table_add (stream_names,
+                        (void *)modulemd_module_stream_get_stream_name (
```

Comment:
You don't actually need to cast to `(void *)` here, so don't.

Reply:
It was giving an error saying expected void * but given gchar **

Reply:
If it’s getting `char **` there, something is wrong. `get_stream_name()` returns `const gchar *`

Reply:
My bad, I edited the warning message. I'm sleepy.

Reply:
Ah right, it doesn’t like the implicit cast discarding the `const`. That makes sense. Okay, approved.